OMNA/registerpage.html

```diff
@@ -288,6 +288,600 @@
             </table>
         </div>
 
+        <div>
+            <h2>OMA Security</h2>
+            <table id="registries-table" class="table-bordered">
```

Comment:
all of them are unique and are for a use. I will wait to create my second Pull Request to link to these IDs after we have a chat tomorrow.